mitodl/open-discussions
# Remove mappings for Resources and Exercises from OCW_SECTION_TYPE_MAPPING

#### Pre-Flight checklist

#### What are the relevant tickets?
Fixes #3414 

#### What's this PR do?
Removes mappings for `Resources` and `Exercises` from OCW_SECTION_TYPE_MAPPING

#### How should this be manually tested?
Make sure you have imported the following courses:

```python
sync_ocw_courses(
course_prefixes=[
  "PROD/14/14.33/Spring_2012/14-33-economics-research-and-communication-spring-2012", 
  "PROD/18/18.05/Spring_2014/18-05-introduction-to-probability-and-statistics-spring-2014"
  ], 
  blocklist=[], force_overwrite=True, upload_to_s3=False
  )
   ```

Reindex all courses and verify that neither "Resources" nor "Exercises" show up in the "Resource Types" facets in ocw-next resource search.
